### PR TITLE
PrestaShop Live Update - April 2024

### PR DESCRIPTION
**Pull request by kpodemski**

### CODE REVIEW

File: content/news/public-oss-demo/2024/live-update-april-2024.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+layout: post
+title:  "PrestaShop Live Update - April 2024"
+subtitle: "A summary of the fourth Live Update in 2024"
+date:   2024-04-19
+authors: [ kpodemski ]
+icon: icon-video-camera
+tags: [contribute, event, community, live-update]
+---
+
+Meet the PrestaShop Performance project, which aims to enhance loading times and reliability of PrestaShop solution by discovering and tackling key performance issues.
```
Comment (coffrep) [inline content/news/public-oss-demo/2024/live-update-april-2024.md]:
```suggestion
Participants were able to know about the PrestaShop Performance project, which aims to enhance loading times and reliability of the PrestaShop solution by discovering and tackling key performance issues.
```

File: content/news/public-oss-demo/2024/live-update-april-2024.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## PrestaShop Live Update - April 2024
+
+In our fourth live update, our special guest, QA Performance Specialist at PrestaShop, [Elyes Naouar](https://github.com/elyesnaouar) introduced the community to the PrestaShop Performance project. This initiative is focused on identifying, measuring, and addressing performance issues to improve the loading times and reliability of the PrestaShop solution.
```
Comment (coffrep) [inline content/news/public-oss-demo/2024/live-update-april-2024.md]:
```suggestion
In our fourth Live Update, our special guest, [Elyes Naouar](https://github.com/elyesnaouar), QA Performance Specialist at PrestaShop, introduced the community to the PrestaShop Performance project. This initiative is focused on identifying, measuring, and addressing performance issues to improve the loading times and reliability of the PrestaShop solution.
```

File: content/news/public-oss-demo/2024/live-update-april-2024.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+You can download the slides from Elyes presentation [here](https://drive.google.com/file/d/1pyPUetjxdRyKXdIuqA7SHjaWJkWw5Zii/view?usp=sharing).
```
Comment (coffrep) [inline content/news/public-oss-demo/2024/live-update-april-2024.md]:
```suggestion
You can download the slides from Elyes' presentation [here](https://drive.google.com/file/d/1pyPUetjxdRyKXdIuqA7SHjaWJkWw5Zii/view?usp=sharing).
```

**Review by coffrep — APPROVED** [review]
Hi @kpodemski this is a great blog post for a very interesting Live Update. I'm approving with edits.

**Comment by kpodemski** [comment]
thanks @coffrep !

**Pull request merged**